Recrutify/Recrutify.Services/Validators/BaseProjectValidator.cs

```diff
@@ -48,25 +53,37 @@ protected BaseProjectValidator()
                  .NotEmpty();
             RuleForEach(p => p.Mentors)
                 .NotNull()
-                .NotEmpty();
+                .NotEmpty()
+                .CustomAsync(CheckStuffAsync);
```

Comment:
use CustomAsync(CheckStuffAsync) only once for  RuleFor(p => p) and check by ids of all Mentors, Managers, Interviewers, Recruiters together

Reply:
St**a**ff